site/en/blog/deps-rems-89/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Deprecations and removals in Chrome 89
+description: >
+  A round up of the deprecations and removals in Chrome 89 to help you plan.
+layout: 'layouts/blog-post.njk'
+date: 2021-01-28
+authors:
+  - joemedley
+hero: './dep-rem_480.jpg'
```

Comment:
Hey @jpmedley for the hero image would you mind uploading it to our image cdn ([guide](https://developer.chrome.com/docs/handbook/how-to/add-media/)) and copy the image into here.

Here's [an example](https://github.com/GoogleChrome/developer.chrome.com/blame/d0275dc7c05ffaad4680a5b52f95928a46b67aa4/site/en/blog/new-in-chrome-54/index.md#L13) of what it should look like from one of Pete's posts.

I have a PR that I'm working on which explains the process better but I don't think it'll be up until tomorrow.